project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss

```diff
@@ -24,4 +24,39 @@
   }
 }
 
+#block-homepageabouttheredressboard {
+  display: flex;
+  text-align: left;
+  padding: 4rem 0 5rem;
+  border-bottom: 1px solid $brand-secondary;
+
+  .left {
+    flex: 2;
+  }
+
+  .right{
```

Comment:
Should have a space between right and the opening bracket.

Reply:
Thanks Adam , I have now fixed this